standards/git-commands.md

```diff
@@ -1,22 +1,91 @@
 # Git Commands
+
 Git is a distributed version-control system for tracking changes in source code during software development. It is designed for coordinating work among programmers, but it can be used to track changes in any set of files. Its goals include speed, data integrity, and support for distributed, non-linear workflows.
 
 At Bitwise, we use git for a majority of our software projects.
 
 ## Bitwise Git specific resources
+
 1. [Commits](commits.md)
 2. [Branching](branching.md)
 3. [Code Versioning](code-versioning.md)
 
 ### Lesser known Git Commands
 
+#### Git Switch
+
+Git Official Documentation: https://git-scm.com/docs/git-switch
+
+Git Switch (and Restore) were added to address overloading of the `checkout` command, and to provide alternative (and clearer) ways to do some basic work.
+These commands do not provide new functionality, but may be easier to remember how to use them and what they do.
+
+Switch is useful to switch to another branch, creating a new branch in the process if desired.
+
+```shell
+$ git switch <branch-name>
+```
+
+can be used to switch to an existing branch instead of the checkout analog:
+
+```shell
+$ git checkout <branch-name>
+```
+
+For creating new branches, then switching to them, `switch -c` may seem more intuitive than `checkout -b`.
+
+```shell
+$ git switch -c <branch-name>
+```
+
+can be used to create a new branch and switch to it. Alternatively,
+
+```shell
+$ git checkout -b <branch-name>
+```
+
+provides identical functionality.
+
+#### Git Restore
+
+Git Official Documentation: https://git-scm.com/docs/git-restore
+
+Git Switch (and Restore) were added to address overloading of the `checkout` command, and to provide alternative (and clearer) ways to do some basic work.
+These commands do not provide new functionality, but may be easier to remember how to use them and what they do.
```

Comment:
@cmcoatney How do you feel about making this paragraph unique compared to the Git Switch paragraph above? It looks like line 19-20 is the same as line 52-53.